### PR TITLE
chore: release 0.10.0-beta

### PR DESCRIPTION
**Pull request by ianbotsf**
## Issue \#

(none)

## Description of changes

Release **0.10.0-beta**.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## [0.10.0-beta] - 01/06/2022
+
+**WARNING: Beta releases may contain bugs and no guarantee is made about API stability. They are not recommended for production use!**
+
+### Breaking changes
+* Update codegen to generate base exception rather than UnknownServiceErrorException [#484](https://github.com/awslabs/aws-sdk-kotlin/pull/484)
```
Comment (kggilmer) [inline CHANGELOG.md:8]: nit 

'base *service* exception' would be more descriptive

File: gradle.properties [hunk 2]
```diff
@@ -6,13 +6,13 @@ kotlin.native.ignoreDisabledTargets=true
...
 
 # sdk
-sdkVersion=0.9.6-SNAPSHOT
+sdkVersion=0.10.0-beta
 
 # codegen
 smithyVersion=1.13.1
 smithyGradleVersion=0.5.3
 # smithy-kotlin codegen and runtime are versioned together
-smithyKotlinVersion=0.7.5-SNAPSHOT
```
Comment (kggilmer) [inline gradle.properties:15]: hmm, seems a little weird that smithy-kotlin is keeping same version but crt kotlin is getting bumped to next.

**Review by kggilmer — APPROVED** [review]

File: gradle.properties (hunk 2 above)
Comment (kggilmer) [reply gradle.properties:15]: nm verified against other PRs, LGTM :+1:

Comment (ianbotsf) [reply gradle.properties:15]: The last release never bumped the version of **aws-crt-kotlin** to the next snapshot version so we also had to jump the version in [that release PR](https://github.com/awslabs/aws-crt-kotlin/pull/43/files).

File: CHANGELOG.md (hunk 1 above)
Comment (ianbotsf) [reply CHANGELOG.md:8]: Agreed. These lines are autogenerated from commit messages so the ideal place to fix this would've been there.

**Pull request merged**